Userland/Services/WebContent/ConnectionFromClient.cpp

```diff
@@ -546,10 +546,30 @@ void ConnectionFromClient::inspect_dom_node(u64 page_id, i32 node_id, Optional<W
             return builder.to_byte_string();
         };
 
+        auto serialize_fonts_json = [](Web::DOM::Element& element) -> ByteString {
+            auto properties = element.computed_css_values();
```

Comment:
If you pass the style properties in (like how `serialize_json()` works) then you can get this to work for pseudo-elements pretty easily too. You'll want to rebase on master because it looks like the changes I made for that weren't around when you made this branch.